mission_control/navigator_missions/nav_missions/detect_deliver.py

```diff
@@ -54,11 +47,12 @@ def get_waypoint(self):
     @txros.util.cancellableInlineCallbacks
     def circle_search(self):
```

Comment:
Whenever lqrrt supports built in circle moves, this code will likely change. I will provide examples of how to use the new still when it comes out!